src/app/client/src/app/modules/groups/components/group-members/group-members.component.ts

```diff
@@ -61,10 +61,6 @@ export class GroupMembersComponent implements OnInit, OnDestroy {
           this.addTelemetry('member-card-menu-close');
         }
       });
-    this.groupsService.showLoader.subscribe(showLoader => {
-      this.showLoader = showLoader;
-    });
-
     this.groupsService.membersList.subscribe(members => {
```

Comment:
Why do we need these sunbscribe, when ther eis not api call itself

Reply:
these subscriptions are not related to API . these are events to detect the changes. please have a look at groups.service.ts.